taxcalc/utils.py

```diff
@@ -86,7 +88,7 @@
 # labels list to map a label to the correct column in a difference table.
 
 DIFF_VARIABLES = ['expanded_income', 'c00100', 'aftertax_income',
-                  'iitax', 'payrolltax', 'combined', 's006']
+                  'iitax', 'payrolltax', 'combined', 's006','ubi']
```

Comment:
One of the reasons the tests are failing is this line. You need to add a space between the coma and 'ubi' to comply with PEP8 standards.